Relazione/RelazioneTecWeb.tex

```diff
@@ -71,6 +71,30 @@
 			\subsubsection{Amministratore}
 	\section{Progettazione}
 		\subsection{Obiettivi}
+		Gli obiettivi principali perseguiti durante la progettazione del sito sono i seguenti:
+		\begin{itemize}
+		    \item \textbf{Separazione tra struttura, presentazione e comportamento:}
+		    Obiettivo fondamentale, in quanto raggiungerlo permette di soddisfare più agevolmente anche gli altri punti. 
+		    La struttura è stata realizzata con documenti in XHTML 1.0 Strict dove possibile, così da garantire una maggiore retrocompatibilità con vecchi browsers, ed HTML5 dove si sono ritenute necessarie le funzionalità aggiuntive permesse dal linguaggio. Tutto il codice redatto è stato scritto secondo le raccomandazioni W3C, accertando che fossero state rispettate tramite validazione. 
+		    La presentazione è stata sviluppata con fogli di stile CSS linkati, mentre il comportamento con script esterni realizzati in Javascript e PHP. In questo modo la struttura non dovrà cambiare, anche a seguito di modifiche alla presentazione del sito.
+		    \item \textbf{Accessibilità:}
+		    Il sito deve poter essere fruibile agevolmente dal maggior numero di utenti possibile, compresi quelli con differenti tipi di disabilità. Per garantire una buona accessibilità, alcune misure adottate sono:
+		    \begin{itemize}
+		        \item Uso dei tabindex nel menù di navigazione;
+		        \item Testo alternativo per le immagini;
+		        \item Testi e link con buoni livelli di contrasto;
+		        \item Uso dell'attributo lang per testi non in italiano;
+		    \end{itemize}
+		    \item \textbf{Fluidità:}
+		    Il sito deve poter essere consultabile tramite varie tipologie di dispositivi, tra cui PC desktop, tablet e smartphone. Bisogna quindi garantire una buona adattabilità alle differenti dimensioni di schermo.
+		    \item \textbf{Fruibilità:}
+		    Per realizzare un sito navigabile intuitivamente si sono seguite alcune linee guida comuni nel web, come ad esempio:
+		    \begin{itemize}
+		        \item Sfruttare un layout a tre pannelli (per riferimento vedere il capitolo 3.2 Layout METTERE UN RIFERIMENTO?);
```

Comment:
Noi usiamo il layout a 3 pannelli? Perchè noi non abbiamo il menu a sinistra ma sopra e non so se si chiami sempre così. Prova a vedere cosa trovi su internet e cerco un'attimo anche io

Reply:
Azz vero, mi sa che deve avere la barra laterale per chiamarsi così, dopo tolgo

Reply:
Se riesci a trovare il nome corretto metti quello

Reply:
Da quello che ho trovato online il nostro dovrebbe chiamarsi layout ad una colonna, ma secondo me per il contesto di questa frase è meglio restare generali e dire che semplicemente bisogna usare un layout che organizzi bene il contenuto per non far fare confusione all'utente... Volendo dopo che abbiamo fatto la sezione per il layout si può riguardare se conviene cambiare la frase in base a quello che scriviamo la. Ti va bene?

Reply:
Ok